ebodac/src/main/java/org/motechproject/ebodac/template/PdfReportCTemplate.java

```diff
@@ -11,6 +12,8 @@
 
     public PdfReportCTemplate(OutputStream outputStream) {
         super(TEMPLATE_PATH, FIRST_PAGE_RECTANGLE, outputStream);
+        PdfReportCTemplate.TABLE_FONT = new Font(Font.FontFamily.HELVETICA, 6);
```

Comment:
I know, that's why I am wondering on the opinon of users on these reports. For the long-term , the only way I see is that we should define clearly the max amount of columns for a PDF report and only add that amount for 8px max.

For the sort term, I would like to see the reaction of users due to we are not broking anything, the visualization is there it is a matter of accepance from Users, that is what I wonder with this PR.